deployment/run-with-kind.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This script utilize KIND (https://github.com/kubernetes-sigs/kind), to install Presidio
+# into docker (with k8s emulation).
+# This script is intended to get Presidio up and running in no time, so you can play with it.
+# However it is NOT intended to be installed in this manner for any workload (dev or prod).
+
+# Clone KIND and build it (this way we eliminate the need to have GO installed)
```

Comment:
not sure this is the best approach over having kind already installed. keeping this to see what other think.
downside is that we build kind every time.

Reply:
I'm thinking about it from a new user perspective. 
How many of those potential users have Docker and how many have Kind.
If you have kind, you also have the presidio chart...just install it.

otherwise, we can automate it.